docs/reference/ilm/policy-definitions.asciidoc

```diff
@@ -367,7 +367,7 @@ index "my_index" must be the write index for the alias. For more information, re
 
 [source,console]
 --------------------------------------------------
-PUT my_index
+PUT my_index-000001
```

Comment:
The preceding paragraph refers to the "my_index" index. We should update that reference as well.